packages/text-input-react/README.md

```diff
@@ -2,9 +2,49 @@
 
 ## Beskrivelse
 
-[Bruk og prinsipper TextField](https://fremtind.github.io/jokul/components/textfield/) er beskrevet i `@fremtind/jkl-text-input` stil-pakken.
-[Bruk og prinsipper TextArea](https://fremtind.github.io/jokul/components/textarea/) er beskrevet i `@fremtind/jkl-text-input` stil-pakken.
+[Bruk og prinsipper](https://fremtind.github.io/jokul/components/textfield/) er beskrevet i `@fremtind/jkl-text-input` stil-pakken.
 
 ## Kom i gang
 
 [Lær hvordan du kan ta i bruk Jøkul](https://fremtind.github.io/jokul/developer/getting-started/)
+
+## Bruk av React-pakken
+
+### Installasjon
+
+1. Installèr pakken med `yarn add @fremtind/jkl-text-input-react` eller `npm i @fremtind/jkl-text-input-react`. Stil-pakken blir automatisk installert som en avhengighet.
+
+2. Importer _både_ React-pakken (velg komponenten(e) du trenger) og stil-pakken i prosjektet ditt:
+
+```js
+import { TextField, TextArea, ActionTextField, InlineTextField, BaseInputField } from "@fremtind/jkl-text-input-react";
+import "@fremtind/jkl-text-input/text-input.min.css";
+```
+
+### Generell Bruk
+
+Du må spesifisere en `label` for samtlige komponenter omtalt under "Generell Bruk".
```

Comment:
Kanskje omformulere denne litt siden variantene ikke listes opp lenger. Og kanskje spesifisere at `label` ikke vises for `InlineTextField`, men fortsatt brukes for skjermlesere? (bare så folk ikke er forvirra over at den er påkrevd, men ikke synes)